zbus/src/abstractions/process.rs

```diff
@@ -78,6 +78,101 @@ impl Command {
     }
 }
 
+/// A wrapper around the child API of the underlying async runtime.
+pub(crate) struct Child(
+    #[allow(unused)]
+    #[cfg(not(feature = "tokio"))]
+    async_process::Child,
+    #[allow(unused)]
+    #[cfg(feature = "tokio")]
+    tokio::process::Child,
```

Comment:
Do we really need a new type here? AFAICT the APIs are identical, as evident from the `take_stdio` impl that identical for both backends. Moreover, you still end up with different `ReadHalf` and `WriteHalf` impls for each backend in the end, so we don't gain anything from this abstraction.